core/tests/protractor_desktop/profileFeatures.js

```diff
@@ -110,3 +114,60 @@ describe('Customized profile page for current user', function() {
     general.checkForConsoleErrors([]);
   });
 });
+
+describe('View explorations created by user in their profile page',
+  function() {
+    var TEST_USERNAME = 'myUser';
+    var TEST_EMAIL = TEST_USERNAME + '@example.com';
+
+    var ANOTHER_USERNAME = 'anotherUser';
+    var ANOTHER_EMAIL = ANOTHER_USERNAME + '@example.com';
+
+    var profilePage = null;
+    var creatorDashboardPage = null;
+
+    var EXPLORATION = {
+      title: 'A new exploration',
+      category: 'Learning',
+      objective: 'The goal is to create a new exploration',
+      language: 'English'
+    };
+
+    beforeAll(function() {
+      profilePage = new ProfilePage.ProfilePage();
+      creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
+    });
+
+    it('another user should create a new exploration', function() {
```

Comment:
it block description should always begin with `should` or `should not`

Reply:
I've fixed it

Reply:
Done